code/game/objects/structures/depot.dm

```diff
@@ -12,8 +12,12 @@
 /obj/structure/fusionreactor/Initialize()
 	. = ..()
 	depotarea = areaMaster
-	if(depotarea)
+	if(istype(depotarea))
 		depotarea.reactor = src
+		spawn(50) // Necessary since random spawners may take a second to spawn turrets in depot.
```

Comment:
Previously, turrets were using "var/area/syndicate_depot/core/depotarea = areaMaster" every time they had to call depotarea for something, which was... not great. It also did not work outside the /core area, which meant that external turrets in the sand area couldn't use depot AI functions. In practice, this meant that anyone who died to the outside turrets wouldn't actually trigger the depot's AI to think a run had been started. This matters because it leads to runs that wipe on the first turret not even showing up in the admin logs for depot runs. Future updates to depot AI are going to need more reliable tracking for when depot runs start/finish, so this fix is also partly in preparation for that. This has to use spawn() because turrets may not spawn in instantly at roundstart, and it has to use range() because the turrets may be in a different area.

Reply:
this, makes me kind of uncomfy. let me look through our systems and pick out something nicer to work with.

Reply:
Alright: so, in this item's `Initialize`, return `INITIALIZE_HINT_LATELOAD` instead of returning the value of the parent call (but still do the parent call).

Then, add a `LateInitialize` override to this atom to do the ranged sweep for those syndicate turrets.